src/models/network/evm/named_network.rs

```diff
@@ -123,17 +122,20 @@ pub enum EvmNamedNetwork {
     #[strum(to_string = "mantle")]
     #[serde(alias = "mantle")]
     Mantle = 5000,
-    #[strum(to_string = "mantle-testnet")]
-    #[serde(alias = "mantle-testnet")]
-    MantleTestnet = 5001,
```

Comment:
This one is deprecated? I noticed it has a different chain id then mantle sepolia